zmk/Configure.mk

```diff
@@ -39,6 +39,16 @@ Configure.Options ?=
 # location. One place where this happens is zmk test suite.
 ifneq (,$(Project.Name))
 -include config.$(Project.Name).mk
+# Those are copied from z.mk and need to be in sync.
```

Comment:
Would include be feasible here?

Reply:
Include as in `include` without the leading `-`? No because this code runs both when the project is configured and when it is not configured, and the `-` allows us to silently ignore the missing configuration file.

Reply:
I was thinking of putting shared part into separate file and include it in both z.mk and here

Reply:
Ah, I see what you mean now.

I would not do it yet simply because that file is a bit tiny and we really want to execute the code again, not just include a "module" like all of zmk is doing (unlike the native make include system).

I'm still a bit unhappy about how this looks like, that we need this repetition, and that the usual make functional definitions cannot be used here easily.